mojo/public/cpp/bindings/lib/connector.cc

```diff
@@ -624,9 +635,8 @@ void Connector::ReadAllAvailableMessages() {
 
     switch (rv) {
       case MOJO_RESULT_OK:
-        if (!DispatchMessage(std::move(message)) || !weak_self || paused_) {
+        if (!DispatchMessage(std::move(message)) || !weak_self || paused_)
```

Comment:
Why are you converting if multi-line blocks to single-line? I have seen this a few times now. Not sure if intended?